R/github.R

```diff
@@ -23,16 +23,51 @@ github_response <- function(req) {
   parsed
 }
 
-github_GET <- function(path, ..., pat = github_pat()) {
+github_GET <- function(path, ..., pat = github_pat(),
+                       host = "api.github.com") {
```

Comment:
You'll need to update this for our recent tweaks which include the protocol in the host string